cmd/wallet-sdk-js/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "wallet-sdk",
+  "version": "0.0.1",
+  "description": "Implements a simplified interface to interop with JS for wallet-sdk.",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "karma start --single-run"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/trustbloc/wallet-sdk.git"
+  },
+  "keywords": [
+    "wallet",
```

Comment:
May wanna use [these](https://github.com/trustbloc/wallet/blob/main/package.json#L18-L21) for better visibility